<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 7: Applications of ddPCR in Genomic Analysis | Statistics for ddPCR</title>
  <meta name="description" content="<p>This is a minimal example of using the bookdown package to write a book.
set in the _output.yml file.
The HTML output format for this example is bookdown::gitbook,</p>" />
  <meta name="generator" content="bookdown 0.42 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 7: Applications of ddPCR in Genomic Analysis | Statistics for ddPCR" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="<p>This is a minimal example of using the bookdown package to write a book.
set in the _output.yml file.
The HTML output format for this example is bookdown::gitbook,</p>" />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 7: Applications of ddPCR in Genomic Analysis | Statistics for ddPCR" />
  
  <meta name="twitter:description" content="<p>This is a minimal example of using the bookdown package to write a book.
set in the _output.yml file.
The HTML output format for this example is bookdown::gitbook,</p>" />
  

<meta name="author" content="Kyungmin In, GCCL" />


<meta name="date" content="2025-06-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="chapter-6-confidence-intervals-and-limit-of-detection-lod.html"/>
<link rel="next" href="chapter-8-quality-control-and-best-practices-in-ddpcr.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Statistics for ddPCR</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction to Digital PCR (ddPCR)</a></li>
<li class="chapter" data-level="" data-path="author.html"><a href="author.html"><i class="fa fa-check"></i>Author</a></li>
<li class="chapter" data-level="" data-path="chapter-1-bernoulli-trial.html"><a href="chapter-1-bernoulli-trial.html"><i class="fa fa-check"></i>Chapter 1: Bernoulli Trial</a></li>
<li class="chapter" data-level="" data-path="chapter-2-binomial-distribution.html"><a href="chapter-2-binomial-distribution.html"><i class="fa fa-check"></i>Chapter 2: Binomial Distribution</a></li>
<li class="chapter" data-level="" data-path="chapter-3-poisson-distribution.html"><a href="chapter-3-poisson-distribution.html"><i class="fa fa-check"></i>Chapter 3: Poisson Distribution</a></li>
<li class="chapter" data-level="" data-path="chapter-4-estimating-copy-number-in-ddpcr.html"><a href="chapter-4-estimating-copy-number-in-ddpcr.html"><i class="fa fa-check"></i>Chapter 4: Estimating Copy Number in ddPCR</a></li>
<li class="chapter" data-level="" data-path="chapter-5-cpd-and-detection-probability.html"><a href="chapter-5-cpd-and-detection-probability.html"><i class="fa fa-check"></i>Chapter 5: CPD and Detection Probability</a></li>
<li class="chapter" data-level="" data-path="chapter-6-confidence-intervals-and-limit-of-detection-lod.html"><a href="chapter-6-confidence-intervals-and-limit-of-detection-lod.html"><i class="fa fa-check"></i>Chapter 6: Confidence Intervals and Limit of Detection (LoD)</a></li>
<li class="chapter" data-level="" data-path="chapter-7-applications-of-ddpcr-in-genomic-analysis.html"><a href="chapter-7-applications-of-ddpcr-in-genomic-analysis.html"><i class="fa fa-check"></i>Chapter 7: Applications of ddPCR in Genomic Analysis</a></li>
<li class="chapter" data-level="" data-path="chapter-8-quality-control-and-best-practices-in-ddpcr.html"><a href="chapter-8-quality-control-and-best-practices-in-ddpcr.html"><i class="fa fa-check"></i>Chapter 8: Quality Control and Best Practices in ddPCR</a></li>
<li class="chapter" data-level="" data-path="chapter-9-conclusion.html"><a href="chapter-9-conclusion.html"><i class="fa fa-check"></i>Chapter 9: Conclusion</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Statistics for ddPCR</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="chapter-7-applications-of-ddpcr-in-genomic-analysis" class="section level1 unnumbered hasAnchor">
<h1>Chapter 7: Applications of ddPCR in Genomic Analysis<a href="chapter-7-applications-of-ddpcr-in-genomic-analysis.html#chapter-7-applications-of-ddpcr-in-genomic-analysis" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Digital PCR (ddPCR) enables absolute quantification of nucleic acids without the need for calibration curves.<br />
This makes it a powerful tool in genomic applications where precision is critical, such as:</p>
<ul>
<li><strong>Copy Number Variation (CNV) detection</strong></li>
<li><strong>Rare allele or mutation detection</strong></li>
</ul>
<p><strong>Copy Number Variation (CNV) Detection</strong></p>
<p><strong>Copy Number Variation (CNV)</strong> refers to a gain or loss of gene copies in the genome.
ddPCR quantifies the number of gene copies in a sample and compares it to a <strong>reference gene</strong> with known normal copy number (usually 2).</p>
<p>The relative copy number is calculated as:</p>
<p><span class="math display">\[
\text{Copy Ratio} = \frac{\text{Target Gene Concentration}}{\text{Reference Gene Concentration}}
\]</span></p>
<p><strong>Example</strong>:</p>
<div class="sourceCode" id="cb33"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb33-1"><a href="chapter-7-applications-of-ddpcr-in-genomic-analysis.html#cb33-1" tabindex="-1"></a>target <span class="ot">&lt;-</span> <span class="fl">10.1</span> <span class="co"># estimated concentration of target gene (copies/μL)</span></span>
<span id="cb33-2"><a href="chapter-7-applications-of-ddpcr-in-genomic-analysis.html#cb33-2" tabindex="-1"></a>reference <span class="ot">&lt;-</span> <span class="fl">5.0</span> <span class="co"># estimated concentration of reference gene (copies/μL)</span></span>
<span id="cb33-3"><a href="chapter-7-applications-of-ddpcr-in-genomic-analysis.html#cb33-3" tabindex="-1"></a>copy_ratio <span class="ot">&lt;-</span> target <span class="sc">/</span> reference</span>
<span id="cb33-4"><a href="chapter-7-applications-of-ddpcr-in-genomic-analysis.html#cb33-4" tabindex="-1"></a><span class="fu">round</span>(copy_ratio, <span class="dv">2</span>)</span></code></pre></div>
<pre><code>## [1] 2.02</code></pre>
<p><strong>Interpretation</strong></p>
<table>
<thead>
<tr class="header">
<th>Copy Ratio</th>
<th>Interpretation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>~1.0</td>
<td>Normal diploid copy</td>
</tr>
<tr class="even">
<td>~0.5</td>
<td>Heterozygous deletion</td>
</tr>
<tr class="odd">
<td>~2.0</td>
<td>Gene duplication</td>
</tr>
</tbody>
</table>
<p>ddPCR is especially useful for detecting subtle CNVs that may not be distinguishable by qPCR or arrays.</p>
<p><strong>Rare Allele Detection</strong></p>
<p>ddPCR can detect low-frequency mutations, even at &lt; 0.1% mutant allele frequency (MAF). This is crucial for applications such as:</p>
<ul>
<li>Detecting resistance mutations in cancer (e.g., EGFR T790M)</li>
<li>Tracking minimal residual disease (MRD)</li>
<li>Analyzing circulating tumor DNA (ctDNA)</li>
</ul>
<p><strong>Dual-probe strategy</strong></p>
<p>Each droplet is tested for both:</p>
<ul>
<li>Wild-type (WT) using one probe (e.g., HEX)</li>
<li>Mutant (MT) using another probe (e.g., FAM)</li>
<li>The Mutant Allele Frequency (MAF) is calculated as:</li>
</ul>
<p><span class="math display">\[
\text{MAF} = \frac{\text{Mutanat Copies}}{\text{Mutant Copies + Wild-Type Copies}}
\]</span></p>
<p><strong>Example</strong>:</p>
<div class="sourceCode" id="cb35"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb35-1"><a href="chapter-7-applications-of-ddpcr-in-genomic-analysis.html#cb35-1" tabindex="-1"></a>mutant <span class="ot">&lt;-</span> <span class="dv">6</span></span>
<span id="cb35-2"><a href="chapter-7-applications-of-ddpcr-in-genomic-analysis.html#cb35-2" tabindex="-1"></a>wt <span class="ot">&lt;-</span> <span class="dv">994</span></span>
<span id="cb35-3"><a href="chapter-7-applications-of-ddpcr-in-genomic-analysis.html#cb35-3" tabindex="-1"></a></span>
<span id="cb35-4"><a href="chapter-7-applications-of-ddpcr-in-genomic-analysis.html#cb35-4" tabindex="-1"></a>maf <span class="ot">&lt;-</span> mutant <span class="sc">/</span> (mutant <span class="sc">+</span> wt)</span>
<span id="cb35-5"><a href="chapter-7-applications-of-ddpcr-in-genomic-analysis.html#cb35-5" tabindex="-1"></a><span class="fu">round</span>(maf <span class="sc">*</span> <span class="dv">100</span>, <span class="dv">3</span>)</span></code></pre></div>
<pre><code>## [1] 0.6</code></pre>
<p>This demonstrates ddPCR’s ability to detect rare mutations in a background of wild-type DNA.</p>
<p><strong>Summary</strong></p>
<table>
<colgroup>
<col width="21%" />
<col width="31%" />
<col width="48%" />
</colgroup>
<thead>
<tr class="header">
<th>Application</th>
<th>Measurement</th>
<th>ddPCR Advantage</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>CNV Detection</td>
<td>Copy ratio (Target / Reference)</td>
<td>Absolute quantification without a standard curve</td>
</tr>
<tr class="even">
<td>Rare Allele Detection</td>
<td>Mutant Allele Frequency (MAF)</td>
<td>Sensitive detection of &lt;0.1% MAF</td>
</tr>
</tbody>
</table>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="chapter-6-confidence-intervals-and-limit-of-detection-lod.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="chapter-8-quality-control-and-best-practices-in-ddpcr.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": "https://github.com/USERNAME/REPO/edit/BRANCH/07-Application.Rmd",
    "text": "Edit"
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": ["_main.HTML"],
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
