<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 8: Quality Control and Best Practices in ddPCR | Statistics for ddPCR</title>
  <meta name="description" content="<p>This is a minimal example of using the bookdown package to write a book.
set in the _output.yml file.
The HTML output format for this example is bookdown::gitbook,</p>" />
  <meta name="generator" content="bookdown 0.42 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 8: Quality Control and Best Practices in ddPCR | Statistics for ddPCR" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="<p>This is a minimal example of using the bookdown package to write a book.
set in the _output.yml file.
The HTML output format for this example is bookdown::gitbook,</p>" />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 8: Quality Control and Best Practices in ddPCR | Statistics for ddPCR" />
  
  <meta name="twitter:description" content="<p>This is a minimal example of using the bookdown package to write a book.
set in the _output.yml file.
The HTML output format for this example is bookdown::gitbook,</p>" />
  

<meta name="author" content="Kyungmin In, GCCL" />


<meta name="date" content="2025-06-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="chapter-7-applications-of-ddpcr-in-genomic-analysis.html"/>
<link rel="next" href="chapter-9-conclusion.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Statistics for ddPCR</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction to Digital PCR (ddPCR)</a></li>
<li class="chapter" data-level="" data-path="author.html"><a href="author.html"><i class="fa fa-check"></i>Author</a></li>
<li class="chapter" data-level="" data-path="chapter-1-bernoulli-trial.html"><a href="chapter-1-bernoulli-trial.html"><i class="fa fa-check"></i>Chapter 1: Bernoulli Trial</a></li>
<li class="chapter" data-level="" data-path="chapter-2-binomial-distribution.html"><a href="chapter-2-binomial-distribution.html"><i class="fa fa-check"></i>Chapter 2: Binomial Distribution</a></li>
<li class="chapter" data-level="" data-path="chapter-3-poisson-distribution.html"><a href="chapter-3-poisson-distribution.html"><i class="fa fa-check"></i>Chapter 3: Poisson Distribution</a></li>
<li class="chapter" data-level="" data-path="chapter-4-estimating-copy-number-in-ddpcr.html"><a href="chapter-4-estimating-copy-number-in-ddpcr.html"><i class="fa fa-check"></i>Chapter 4: Estimating Copy Number in ddPCR</a></li>
<li class="chapter" data-level="" data-path="chapter-5-cpd-and-detection-probability.html"><a href="chapter-5-cpd-and-detection-probability.html"><i class="fa fa-check"></i>Chapter 5: CPD and Detection Probability</a></li>
<li class="chapter" data-level="" data-path="chapter-6-confidence-intervals-and-limit-of-detection-lod.html"><a href="chapter-6-confidence-intervals-and-limit-of-detection-lod.html"><i class="fa fa-check"></i>Chapter 6: Confidence Intervals and Limit of Detection (LoD)</a></li>
<li class="chapter" data-level="" data-path="chapter-7-applications-of-ddpcr-in-genomic-analysis.html"><a href="chapter-7-applications-of-ddpcr-in-genomic-analysis.html"><i class="fa fa-check"></i>Chapter 7: Applications of ddPCR in Genomic Analysis</a></li>
<li class="chapter" data-level="" data-path="chapter-8-quality-control-and-best-practices-in-ddpcr.html"><a href="chapter-8-quality-control-and-best-practices-in-ddpcr.html"><i class="fa fa-check"></i>Chapter 8: Quality Control and Best Practices in ddPCR</a></li>
<li class="chapter" data-level="" data-path="chapter-9-conclusion.html"><a href="chapter-9-conclusion.html"><i class="fa fa-check"></i>Chapter 9: Conclusion</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Statistics for ddPCR</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="chapter-8-quality-control-and-best-practices-in-ddpcr" class="section level1 unnumbered hasAnchor">
<h1>Chapter 8: Quality Control and Best Practices in ddPCR<a href="chapter-8-quality-control-and-best-practices-in-ddpcr.html#chapter-8-quality-control-and-best-practices-in-ddpcr" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>While digital PCR (ddPCR) provides highly precise quantification, ensuring data quality requires careful experimental design, appropriate controls, and post-run validation.<br />
This chapter outlines essential quality control (QC) considerations and best practices to ensure reliable and reproducible ddPCR results.</p>
<p><strong>1. Droplet Quality and Thresholding</strong>
- <strong>Minimum Droplet Count</strong>:
A minimum of <strong>10,000 accepted droplets per well</strong> is recommended for statistically robust quantification.<br />
Fewer droplets result in wider confidence intervals and increased error.</p>
<ul>
<li><p><strong>Rain Detection</strong>:
“Rain” refers to droplets with intermediate fluorescence that fall between the positive and negative peaks.<br />
These droplets may result from:</p>
<ul>
<li>Poor probe or primer efficiency</li>
<li>Suboptimal thermal cycling</li>
<li>Sample degradation or inhibition</li>
</ul></li>
</ul>
<blockquote>
<p>Inspection and adjusting thresholds may be necessary.</p>
</blockquote>
<p><strong>2. Controls</strong></p>
<ul>
<li><p><strong>No Template Control (NTC)</strong>:
Included in every run to monitor contamination.
Expected result: <strong>0 positive droplets</strong>.</p></li>
<li><p><strong>Positive Control</strong>:
A sample with a known concentration of target DNA, used to confirm assay performance.</p></li>
<li><p><strong>Reference Gene Control</strong>:
In CNV assays, use a <strong>stable reference gene</strong> (e.g., RPP30 or RNaseP) to normalize copy number.</p></li>
</ul>
<p><strong>3. Assessing Reproducibility</strong></p>
<p>Run <strong>technical replicates</strong> to ensure precision and identify potential outliers.</p>
<div class="sourceCode" id="cb37"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb37-1"><a href="chapter-8-quality-control-and-best-practices-in-ddpcr.html#cb37-1" tabindex="-1"></a>replicates <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fl">10.2</span>, <span class="fl">10.4</span>, <span class="fl">9.9</span>)</span>
<span id="cb37-2"><a href="chapter-8-quality-control-and-best-practices-in-ddpcr.html#cb37-2" tabindex="-1"></a><span class="fu">mean</span>(replicates) </span></code></pre></div>
<pre><code>## [1] 10.16667</code></pre>
<div class="sourceCode" id="cb39"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb39-1"><a href="chapter-8-quality-control-and-best-practices-in-ddpcr.html#cb39-1" tabindex="-1"></a><span class="fu">sd</span>(replicates) </span></code></pre></div>
<pre><code>## [1] 0.2516611</code></pre>
<div class="sourceCode" id="cb41"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb41-1"><a href="chapter-8-quality-control-and-best-practices-in-ddpcr.html#cb41-1" tabindex="-1"></a>cv <span class="ot">&lt;-</span> <span class="fu">sd</span>(replicates) <span class="sc">/</span> <span class="fu">mean</span>(replicates) <span class="sc">*</span> <span class="dv">100</span></span>
<span id="cb41-2"><a href="chapter-8-quality-control-and-best-practices-in-ddpcr.html#cb41-2" tabindex="-1"></a><span class="fu">round</span>(cv, <span class="dv">2</span>) <span class="co"># CV (%) &lt; 10% is ideal</span></span></code></pre></div>
<pre><code>## [1] 2.48</code></pre>
<p><strong>4. False Positives and False Negatives</strong></p>
<table>
<colgroup>
<col width="25%" />
<col width="40%" />
<col width="34%" />
</colgroup>
<thead>
<tr class="header">
<th>Error Type</th>
<th>Possible Cause</th>
<th>Mitigation Strategy</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>False Positive</td>
<td>Contamination, misclassified “rain” droplets</td>
<td>Use strict NTC, adjust threshold</td>
</tr>
<tr class="even">
<td>False Negative</td>
<td>Poor droplet generation, failed amplification</td>
<td>Increase input DNA, optimize conditions</td>
</tr>
</tbody>
</table>
<blockquote>
<p>For rare mutation detection, even a single false positive can affect interpretation — validation is essential.</p>
</blockquote>
<p><strong>5. LoD Confirmation via Dilution Series</strong></p>
<p>To confirm the Limit of Detection (LoD), use a serial dilution of a known template and assess the lowest concentration at which detection is consistent.</p>
<ul>
<li>Run 3 or more replicates per dilution level</li>
<li>LoD is typically the lowest level at which all replicates are positive</li>
</ul>
<table>
<thead>
<tr class="header">
<th>Practice</th>
<th>Recommendation</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Droplet Count</td>
<td>&gt;10,000 per well</td>
</tr>
<tr class="even">
<td>NTC</td>
<td>Should have 0 positive droplets</td>
</tr>
<tr class="odd">
<td>Positive Control</td>
<td>Include known copy number sample</td>
</tr>
<tr class="even">
<td>Replicates</td>
<td>Use technical duplicates or triplicates</td>
</tr>
<tr class="odd">
<td>Threshold Inspection</td>
<td>Adjust for rain manually if needed</td>
</tr>
<tr class="even">
<td>LoD Validation</td>
<td>Perform dilution series experiments</td>
</tr>
</tbody>
</table>
<p>By incorporating these best practices, we can minimize experimental variability, reduce false results, and ensure that ddPCR data are reliable and publication-ready.</p>

</div>
            </section>

          </div>
        </div>
      </div>
<a href="chapter-7-applications-of-ddpcr-in-genomic-analysis.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="chapter-9-conclusion.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": "https://github.com/USERNAME/REPO/edit/BRANCH/08-BestPractice.Rmd",
    "text": "Edit"
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": ["_main.HTML"],
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
