<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Statistics for ddPCR</title>
  <meta name="description" content="<p>This is a minimal example of using the bookdown package to write a book.
set in the _output.yml file.
The HTML output format for this example is bookdown::gitbook,</p>" />
  <meta name="generator" content="bookdown 0.42 and GitBook 2.6.7" />

  <meta property="og:title" content="Statistics for ddPCR" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="<p>This is a minimal example of using the bookdown package to write a book.
set in the _output.yml file.
The HTML output format for this example is bookdown::gitbook,</p>" />
  <meta name="github-repo" content="rstudio/bookdown-demo" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Statistics for ddPCR" />
  
  <meta name="twitter:description" content="<p>This is a minimal example of using the bookdown package to write a book.
set in the _output.yml file.
The HTML output format for this example is bookdown::gitbook,</p>" />
  

<meta name="author" content="Kyungmin In, GCCL" />


<meta name="date" content="2025-06-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  

<link rel="next" href="author.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Statistics for ddPCR</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Introduction to Digital PCR (ddPCR)</a></li>
<li class="chapter" data-level="" data-path="author.html"><a href="author.html"><i class="fa fa-check"></i>Author</a></li>
<li class="chapter" data-level="" data-path="chapter-1-bernoulli-trial.html"><a href="chapter-1-bernoulli-trial.html"><i class="fa fa-check"></i>Chapter 1: Bernoulli Trial</a></li>
<li class="chapter" data-level="" data-path="chapter-2-binomial-distribution.html"><a href="chapter-2-binomial-distribution.html"><i class="fa fa-check"></i>Chapter 2: Binomial Distribution</a></li>
<li class="chapter" data-level="" data-path="chapter-3-poisson-distribution.html"><a href="chapter-3-poisson-distribution.html"><i class="fa fa-check"></i>Chapter 3: Poisson Distribution</a></li>
<li class="chapter" data-level="" data-path="chapter-4-estimating-copy-number-in-ddpcr.html"><a href="chapter-4-estimating-copy-number-in-ddpcr.html"><i class="fa fa-check"></i>Chapter 4: Estimating Copy Number in ddPCR</a></li>
<li class="chapter" data-level="" data-path="chapter-5-cpd-and-detection-probability.html"><a href="chapter-5-cpd-and-detection-probability.html"><i class="fa fa-check"></i>Chapter 5: CPD and Detection Probability</a></li>
<li class="chapter" data-level="" data-path="chapter-6-confidence-intervals-and-limit-of-detection-lod.html"><a href="chapter-6-confidence-intervals-and-limit-of-detection-lod.html"><i class="fa fa-check"></i>Chapter 6: Confidence Intervals and Limit of Detection (LoD)</a></li>
<li class="chapter" data-level="" data-path="chapter-7-applications-of-ddpcr-in-genomic-analysis.html"><a href="chapter-7-applications-of-ddpcr-in-genomic-analysis.html"><i class="fa fa-check"></i>Chapter 7: Applications of ddPCR in Genomic Analysis</a></li>
<li class="chapter" data-level="" data-path="chapter-8-quality-control-and-best-practices-in-ddpcr.html"><a href="chapter-8-quality-control-and-best-practices-in-ddpcr.html"><i class="fa fa-check"></i>Chapter 8: Quality Control and Best Practices in ddPCR</a></li>
<li class="chapter" data-level="" data-path="chapter-9-conclusion.html"><a href="chapter-9-conclusion.html"><i class="fa fa-check"></i>Chapter 9: Conclusion</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Statistics for ddPCR</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="header">
<h1 class="title">Statistics for ddPCR</h1>
<p class="author"><em>Kyungmin In, GCCL</em></p>
<p class="date"><em>2025-06-01</em></p>
</div>
<div id="introduction-to-digital-pcr-ddpcr" class="section level1 unnumbered hasAnchor">
<h1>Introduction to Digital PCR (ddPCR)<a href="index.html#introduction-to-digital-pcr-ddpcr" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Digital PCR (<strong>ddPCR</strong>) is a cutting-edge, highly sensitive technique that offers a more precise method for measuring nucleic acids compared to traditional quantitative PCR (qPCR).
It works by partitioning a sample into thousands of individual reactions, effectively counting the number of positive and negative droplets that contain specific DNA or RNA sequences.
This method allows for the absolute quantification of genetic material without the need for a standard curve or external calibration, which is typically required in qPCR.
The ability of ddPCR to quantify gene copy numbers without calibration stems from its unique principle of directly counting individual molecules.
Instead of relying on relative changes in fluorescence intensity, ddPCR uses Poisson statistics to determine the exact number of target DNA copies in a sample based on the fraction of droplets that test positive for the target sequence.
This feature makes ddPCR especially valuable in applications where high precision is needed, such as gene expression analysis, rare allele detection, and the measurement of gene copy number variations (<strong>CNVs</strong>).
Furthermore, ddPCR is not influenced by factors such as amplification efficiency, which can often introduce errors in traditional PCR-based methods.
As a result, ddPCR is considered a gold standard for precise, reproducible measurements of gene copy numbers, providing an unparalleled level of accuracy in quantification.</p>
<p><strong>Poisson Distribution in ddPCR for Quantifying Gene Copy Numbers</strong></p>
<p>One of the key mathematical foundations of ddPCR is the application of the <strong>Poisson distribution</strong>, which is used to model the random distribution of target DNA molecules into individual reaction partitions (droplets).
This distribution allows ddPCR to calculate the number of DNA copies present in a sample without a calibration curve, making it an exceptionally accurate and reliable method for quantification.</p>
<p>In ddPCR, a sample is partitioned into thousands of microdroplets, and each droplet undergoes PCR amplification.
The number of droplets that contain the target DNA is recorded, and using the Poisson distribution, the concentration of the target DNA can be calculated.
The Poisson distribution models the probability of a given number of occurrences (in this case, the number of DNA molecules) within a fixed interval (the volume of each droplet), assuming the events (molecules of DNA) happen independently and randomly.
The formula for the Poisson distribution is:</p>
<p><span class="math display">\[ P(k)= e^{-\lambda} \cdot\frac{\lambda^k}{k!}\]</span></p>
<p>Where:</p>
<ul>
<li><p>P(k): the probability of observing exactly k target DNA molecules in a droplet</p></li>
<li><p><span class="math inline">\(\lambda\)</span>: the expected average number of DNA molecules in a droplet which we aim to estimate</p></li>
<li><p>e: the Euler’s number (approximately 2.71828)</p></li>
<li><p>k: the number of target molecules observed in the droplet</p></li>
</ul>
<p>In ddPCR, the target DNA is typically spread across the droplet population, and after PCR amplification, some droplets will contain no copies of the target, while others will contain one copy or even more .
And by counting the number of droplets with zero copies (<strong>negative droplets</strong>) and the number of droplets with one or more copies (<strong>positive droplets)</strong>, ddPCR uses the Poisson distribution to estimate the λ, which represents the number of target DNA molecules per droplet.</p>
<p>The key relationship here is that the fraction of positive droplets can be related to the Poisson-distributed number of target molecules in the sample, allowing for the direct calculation of the absolute concentration of target DNA (or gene copy number) in the original sample, without the need for external standards or calibration.</p>
<p>The Poisson model assumes that the DNA molecules are distributed randomly and independently into droplets.
The number of positive droplets (those that contain at least one copy of the target) and negative droplets (those with no copies of the target) allows for the estimation of the concentration of the target gene in the sample using the following equation:</p>
<p><span class="math display">\[C= -\frac{d}{v}ln(1-\frac{p}{N})\]</span></p>
<p>Where:</p>
<ul>
<li><p>N: the number of total droplet</p></li>
<li><p>p: the number of positive droplet for the target gene</p></li>
<li><p>v: the volume of a droplet (in micro liter)</p></li>
<li><p>d: the dilution factor applied to dilute the sample from the stock to the well</p></li>
</ul>
<p>This approach allows ddPCR to provide highly accurate, absolute quantification of gene copy numbers in complex samples, even when the target sequence is rare or present in very low quantities, without the need for reference standards or calibration curves.</p>

</div>
            </section>

          </div>
        </div>
      </div>

<a href="author.html" class="navigation navigation-next navigation-unique" aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
  "sharing": {
    "github": false,
    "facebook": true,
    "twitter": true,
    "linkedin": false,
    "weibo": false,
    "instapaper": false,
    "vk": false,
    "whatsapp": false,
    "all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
  },
  "fontsettings": {
    "theme": "white",
    "family": "sans",
    "size": 2
  },
  "edit": {
    "link": "https://github.com/USERNAME/REPO/edit/BRANCH/index.Rmd",
    "text": "Edit"
  },
  "history": {
    "link": null,
    "text": null
  },
  "view": {
    "link": null,
    "text": null
  },
  "download": ["_main.HTML"],
  "search": {
    "engine": "fuse",
    "options": null
  },
  "toc": {
    "collapse": "subsection"
  }
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
